src/Stratis.Bitcoin.Features.Consensus/Rules/IConsensusRules.cs

```diff
@@ -19,6 +19,11 @@ public interface IConsensusRules
         /// </summary>
         IEnumerable<ConsensusRuleDescriptor> Rules { get; }
 
+        /// <summary>
+        /// Keeps track of how much time different actions took to execute and how many times they were executed.
```

Comment:
After you've added a comment here there is no need to have an exact same comment in `ConsensusRules.cs`. Please add inheritdoc for the performance counter in `ConsensusRules.cs`

Reply:
Cool